docs/source/using/accessing-the-openshift-docker-registry.adoc

```diff
@@ -0,0 +1,44 @@
+= Accessing the OpenShift Docker Registry
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
```

Comment:
we should try to have a second level heading (==) here. The pages render better then. I know some other files don't have it either yet, but the aim should be to have it.